packages/gui/src/electron/validateSha256.ts

```diff
@@ -0,0 +1,68 @@
+import { net, ipcMain, IncomingMessage, BrowserWindow } from 'electron';
```

Comment:
we already have implemented hash validation via hook useVerifyURIHash 
we should use same functionality 
if we need it in electron renderer then we can move everything from hook to ipcRenderer event